terminal/kitty/cyberpunk-neon.conf

```diff
@@ -0,0 +1,45 @@
+# Cyberpunk Neon: https://github.com/Roboron3042/Cyberpunk-Neon
+# Author: Theo (theo-grivel)
+
+background            #000b1e
+foreground            #0abdc6
+selection_background  #fcf6e8
```

Comment:
Where did this come from?

Reply:
I use this file for copy the colors...
https://github.com/Roboron3042/Cyberpunk-Neon/blob/master/terminal/termite/config

and I use this manual of kitty...
https://sw.kovidgoyal.net/kitty/conf/#color-scheme

Reply:
But the fcf6e8 (selection_background) colour is nowhere in the colour scheme.

Looking at the kitty config, setting both selection_foreground and selection_background to "none" would reverse the colours. Wouldn't this be desirable?

Reply:
Yep, it's clearly better!